lib/network/call_account_api.js

```diff
@@ -0,0 +1,134 @@
+// eslint-disable-next-line import/order
+const config = require("../config");
+const https = /^http:/.test(config().upload_prefix) ? require('http') : require('https');
+const querystring = require("querystring");
+const Q = require('q');
+const url = require('url');
+const utils = require("../utils");
+const ensureOption = require('../utils/ensureOption').defaults(config());
+
+const {extend, includes, ensurePresenceOf} = utils;
+
+function call_account_api(method, uri, params, callback, options) {
```

Comment:
Very similar to call_api(). Can we re-use call_api and just add the changes before in call_acount_api? 
Also, should we move callback to the end?

Reply:
Refactored to make_request, call_api and call_account_api are now very thin layers over it.